web/src/app/browser/src/keymanEngine.ts

```diff
@@ -191,11 +191,11 @@ export default class KeymanEngine extends KeymanEngineBase<BrowserConfiguration,
     // globe key - are accessible.
     //
     // The `super` call above initializes `keyboardRequisitioner`, as needed here.
-    this.keyboardRequisitioner.cloudQueryEngine.once('unboundregister', () => {
+    this.keyboardRequisitioner.cloudQueryEngine.once('unboundregister', async () => {
```

Comment:
`EventEmitter`, which is what will trigger this function, does not do `await` / `async` stuff.  If you add code _later_ that motivates it, okay... but as is, you're gaining nothing by the `async` / `await` change here.